certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/message/MessageController.java

```diff
@@ -106,4 +118,25 @@ SendMessageResponse sendMessage(
       @PathVariable("messageId") String messageId) {
     return sendMessageService.send(request, messageId);
   }
+
+  @PostMapping("/{messageId}/saveanswer")
+  SaveAnswerResponse saveAnswer(
+      @RequestBody SaveAnswerRequest request,
+      @PathVariable("messageId") String messageId) {
+    return saveAnswerService.save(request, messageId);
+  }
+
+  @PostMapping("/{messageId}/deleteanswer")
```

Comment:
Should this not be a DeleteMapping?

Reply:
Good point, I also update deleteMessage endpoint and made the required changes in Webcert!